src/Autofac/Core/Resolving/IDependencyTrackingResolveOperation.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using Autofac.Core.Resolving.Middleware;
+using Autofac.Core.Resolving.Pipeline;
+
+namespace Autofac.Core.Resolving
+{
+    /// <inheritdoc />
```

Comment:
Comments on why this interface exists would be good I think.

Reply:
Can you check if the summary I've added is enough?

Reply:
You've got an XML Doc issue in there, but more broadly you could describe this as:

```
The interface indicating that an <see cref="IResolveOperation" /> supports dependency tracking. Required by middleware that must understand the dependency tree.
```

Reply:
Okay, addressed